rainfall_project.py

```diff
@@ -1 +1,17 @@
-#filler line
+total_rainfall = 0
+
+list_of_months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Set", "Oct", "Nov", "Dec"]
+
+year = int(input("Enter the number of years: "))
+
+for years in range(year):
+    print(f"Year {years + 1}")
+    for month in range(12):
+        month_data = (float(input(f"Enter the rainfall amount for {list_of_months[month]}: ")))
+        total_rainfall += month_data
+

```

Comment:
It wouldve been nice to add in a line break here as well at the end of the outter loop. As it stands, everything is squashed together and that reduceds readablitlty.

TO add a line break, 

`print()` or if you want to end on at the end of  line being printed `print("Sample text. \n")`

The \n tells the program to insert a new line.